### ISSUE TITLE
Efficacy of Fecal Microbiota Transplantation in 2 Children With Recurrent Clostridium difficile Infection and Its Impact on Their Growth and Gut Microbiome

### ISSUE THREAD

**Issue by Folakunmi21** [waldronlab/BugSigDBcuration#167]
Efficacy of Fecal Microbiota Transplantation in 2 Children With Recurrent Clostridium difficile Infection and Its Impact on Their Growth and Gut Microbiome – Walia Ritu et al – Journal  of Pediatric Gastroenterology and Nutrition

https://journals.lww.com/jpgn/fulltext/2014/11000/efficacy_of_fecal_microbiota_transplantation_in_2.6.aspx

**Comment by SvetlanaUP** [comment]
@Buraah, yes, you are right, this paper does not report statistical analysis and does not provide curatable results. Your effort will be recored. Please select another paper.

**Comment by SvetlanaUP** [comment]
Not curatable.
